src/test/java/core/basesyntax/service/operation/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.operation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidOperationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationHandlerTest {
+    private static final String DEFAULT_FRUIT_NAME = "apple";
+    private static final Integer INITIAL_FRUIT_QUANTITY = 20;
+    private static final Integer VALID_REQUIRED_FRUIT_QUANTITY = 10;
+    private static final Integer OVERWHELMING_REQ_FRUIT_QUANTITY = INITIAL_FRUIT_QUANTITY + 1;
+    private static final Operation BALANCE_OPERATION = Operation.BALANCE;
+    private static final Operation PURCHASE_OPERATION = Operation.BALANCE;
```

Comment:
PURCHASE_OPERATION = Operation.BALANCE; is it expected?

Reply:
Sorry) It's my bad!